docs/proposed/psa-driver-wrappers-codegen-migration-guide.md

```diff
@@ -28,5 +28,5 @@ While that is the larger goal, for version 1.0 here's what's changed
 #### What's changed
 
 (1) psa_crypto_driver_wrappers.c will from this point on be auto generated.
-(2) The auto generation is based on the template file at scripts/codegen_driverwrappers/psa_crypto_driver_wrappers.conf.
+(2) The auto generation is based on the template file at scripts/data_files/driver_templates/psa_crypto_driver_wrappers.conf.
```

Comment:
A new directory for  driver wrapper templates was created under scripts/data_files. We expect the number of templates  to grow  so would make more sense to keep it organized in a folder and freeze the folder structure now.

Reply:
Would it make sense to rename that directory to something more indicative of the type of files it contains? Like code_templates, or something similar?

Reply:
They are driver-related, so `driver_templates` is more precise than `code_templates`.